### ISSUE TITLE
Turbine: Remote running of turbine. It is not clear you need to upload the simulation (json file).

### ISSUE THREAD

**Issue by fsoepyan** [CCSI-Toolset/FOQUS#27]
From JIRA:
- Reporter: Miguel Zamarripa (@MAZamarripa)
- Assignee: John Eslick (@eslickj)
- Original title: Remote running of turbine. It is not clear you need to upload the simulation (json file).
- Created: 11/Aug/16 9:08 AM
- Updated: 11/Aug/16 9:08 AM

Description:

When switching turbine from "local" to "remote" the user must upload the simulation again. But, we need a "warning" before launching the simulations from "UQ module" to "turbine remote", otherwise they will run without a simulation file.

**Comment by boverhof** [comment]
I'd like to add the ability to "sync" a users simulations via git, rather than doing HTTP PUTs.  This will be targeted for the September milestone.

**Comment by ksbeattie** [comment]
@boverhof is working on this.  It could still be an issue for local running.

**Comment by boverhof** [comment]
Unless I write up a new Data management layer this is not a Turbine issue.  This is a FOQUS issue.

**Comment by MAZamarripa** [comment]
I would say this is more a documentation issue.

**Comment by ksbeattie** [comment]
@MAZamarripa could you make a suggestion on how and where to change the documentation for this?

**Comment by ksbeattie** [comment]
@eslickj will add the above described pop-up warning window.

@ksbeattie will follow up with @MAZamarripa about the additional documentation.

**Comment by MAZamarripa** [comment]
@ksbeattie I will add a brief description once @eslickj is finished with the pop-up window. However, later someone has to run this (successfully) and then type up the documentation. (@fsoepyan maybe, I suggest Bo Hong, since he will be using the parallel simulations for his project).

**Comment by eslickj** [comment]
PR #349

**Comment by eslickj** [comment]
The pop up is in.  @fsoepyan you want to take a look at the docs on this one?

**Comment by fsoepyan** [comment]
@eslickj, can you please let me know if PR #355 and #358 are acceptable documentations?

**Comment by MAZamarripa** [comment]
@fsoepyan the PRs description could be improved.

review section 2.12. Tutorial Using a Remote Turbine Instance (it shows how to set up the configuration file, but not a complete run).
- review item 7 (add a figure, add description, ...,). Item 7 sends you back to the general description of Turbine (2.2.3 and 2.9).

**Comment by fsoepyan** [comment]
Once Remote Turbine is ready, and once I feel comfortable I can use it by myself, I will add more details for the documentation.

**Comment by ksbeattie** [comment]
Waiting for a test of this with new remote turbine

**Comment by ksbeattie** [comment]
@eslickj will work with @fsoepyan on how to run a remote turbine to test this.  @fsoepyan will update the documentation thereafter (clarifying what it meant by "local" and "remote" turbine).

**Comment by fsoepyan** [comment]
Status update: please see #321

**Comment by fsoepyan** [comment]
- PR #471 should take care of the documentation for running turbine remotely on your own computer, if for example, your computer has multiple processors.
- I believe that we are 1 step closer to resolving this issue.
- Once we can run FOQUS simulations in parallel using AWS and/or on multiple computers, the next step would be to update the instructions on how to do that.

**Comment by ksbeattie** [comment]
@fsoepyan says that the next step, as mentioned above, is to add instructions on how to use remote turbine with AWS.

**Comment by fsoepyan** [comment]
Because this issue is about the documentation for Remote Turbine, I will close this issue, as PR #471 has been merged.

However, I will create a new follow-up issue regarding updating the documentation for running Remote Turbine with AWS, if needed.
